drivers/pds/api/dataServiceDeployment.go

```diff
@@ -199,7 +199,7 @@ func (ds *DataServiceDeployment) DeleteDeployment(deploymentID string) (*status.
 	if err != nil {
 		return nil, fmt.Errorf("Error in getting context for api call: %v\n", err)
 	}
-	res, err := dsClient.ApiDeploymentsIdDelete(ctx, deploymentID).Execute()
+	res, err := dsClient.ApiDeploymentsIdDelete(ctx, deploymentID).Force("true").Execute()
```

Comment:
Let us not force deletion. We will stick more closer to what UI Does

Reply:
done